client/locales/en.json

```diff
@@ -7898,4 +7892,4 @@
   "users.troubleSigningIn": {
     "defaultMessage": "Trouble signing in?"
   }
-}
```

Comment:
Trailing newline in en and zh jsons shouldn't be removed.

Reply:
this is not the trailing newline. If you look into the file directly, you'll notice that the newline is still there

Reply:
👍 oh it's not removed, it's added